packages/malabi/src/instrumentation/index.ts

```diff
@@ -46,3 +86,26 @@ export const instrument = () => {
         tracerProvider,
     });
 };
+
+/**
+ * A wrapper that handles creating a span per test run. returns the spans that were created ready for assertion.
+ * @category Main Functions
+ * @param callback an async function containing all of the current test's span generating operations(API calls etc)
+ */
+export const malabi = async (callback): Promise<TelemetryRepository> => {
+    return new Promise((resolve) => {
+        const tracer = trace.getTracer('malabiManualTracer');
```

Comment:
send package name of malabi as in package.json "name", and set the version if possible

Reply:
Added this as an issue for later on https://github.com/aspecto-io/malabi/issues/27